api/task.go

```diff
@@ -131,11 +131,12 @@ type Task struct {
 	TaskCheckRunList []*TaskCheckRun `jsonapi:"relation,taskCheckRun"`
 
 	// Domain specific fields
-	Name              string     `jsonapi:"attr,name"`
-	Status            TaskStatus `jsonapi:"attr,status"`
-	Type              TaskType   `jsonapi:"attr,type"`
-	Payload           string     `jsonapi:"attr,payload"`
-	EarliestAllowedTs int64      `jsonapi:"attr,earliestAllowedTs"`
+	Name    string     `jsonapi:"attr,name"`
+	Status  TaskStatus `jsonapi:"attr,status"`
+	Type    TaskType   `jsonapi:"attr,type"`
+	Payload string     `jsonapi:"attr,payload"`
+	// This field is stored in UTC+0000
```

Comment:
I guess it's OK to not mention this since our convention is always storing the allowed ts in UTC